### PR TITLE
Chrisjurich/remove chem db

### PR DESCRIPTION
**Pull request by chrisjurich**
Removing the last traces of the `load_from_db()` calls from the `enzy_htp.chemical.db` submodule. Tests/installation should be possible now.

### CODE REVIEW

File: enzy_htp/chemical/residue.py [hunk 1]
```diff
@@ -33,46 +230,5916 @@
 
 RESIDUE_CONNECTIVITY_MAP: Dict[str,
                                Dict[str,
-                                    List[str]]] = load_from_db("RESIDUE_CONNECTIVITY_MAP")
+                                    List[str]]] = {
+        "ALA": {
```
Comment (shaoqx) [inline enzy_htp/chemical/residue.py:234]: Is it possible to not make it so long? the original format looks fine to me

Comment (shaoqx) [reply enzy_htp/chemical/residue.py:234]: Oh just found out it was like this in the json. Maybe you can refer to AmberMaps

File: enzy_htp/chemical/residue.py [hunk 2]
```diff
@@ -6,20 +6,217 @@
...
+        "MET": "M",
+        "PHE": "F",
+        "PRO": "P",
+        "SEC": "U",
+        "SER": "S",
+        "THR": "T",
+        "TRP": "W",
+        "TYR": "Y",
+        "VAL": "V"
+} #TODO(shaoqz): @imp2 add related to canonical in the name
```
Comment (shaoqx) [inline enzy_htp/chemical/residue.py:61]: I added several alternative names for some residues. Please add it back

File: test/chemical/test_residue.py [hunk 3]
```diff
@@ -15,12 +15,12 @@ def test_variable_checks():
     assert len(res.AA_LIST) == 21
     assert all_caps(res.AA_LIST)
 
-    assert len(res.THREE_LETTER_AA_MAPPER) == 29
```
Comment (shaoqx) [inline test/chemical/test_residue.py:18]: It should be 29 after those added names are back

**Review by shaoqx — CHANGES_REQUESTED** [review]
Nice! Just one minor change is requested.

File: enzy_htp/chemical/residue.py (hunk 1 above)
Comment (chrisjurich) [reply enzy_htp/chemical/residue.py]: Updated this

File: enzy_htp/chemical/residue.py (hunk 2 above)
Comment (chrisjurich) [reply enzy_htp/chemical/residue.py]: Added extra codes back in.

File: test/chemical/test_residue.py (hunk 3 above)
Comment (chrisjurich) [reply test/chemical/test_residue.py]: Updated.

**Review by shaoqx — APPROVED** [review]
Good job!

**Pull request merged**